app/Shell/Docker.php

```diff
@@ -46,6 +46,21 @@ public function stopContainer(string $containerId): void
         }
     }
 
+    public function logContainer(string $containerId): void
+    {
+        if (! $this->stoppableTakeoutContainers()->contains(function ($container) use ($containerId) {
+            return $container['container_id'] === $containerId;
+        })) {
+            throw new DockerContainerMissingException($containerId);
```

Comment:
Does this give a clear error? Or should we catch it in the command and present a cleaner error?

Reply:
For me is super clear and it isn't necessary a specific exception 🙂. 
The message is: "Docker container 123456 not found." and it is correct because you can log only an active container 
If you feel more comfortable I can provide a specific exception.

Reply:
Nope, that's great! I just hadn't tested it :)